index.html

```diff
@@ -18,31 +18,94 @@
         <div class="header">
 
         	<!-- Insert code for Sign Up, Sign In button here. Add code for "ScriBBler" heading and the sub-heading -->
+	    <input class="headerButtons" type="button" onclick="toggleModal('signUpModal');" value="Sign Up">
+            <input class="headerButtons" type="button" onclick="toggleModal('signInModal');" value="Sign In">
+            <a class="pageTitle">ScriBBler</a>
+            <p class="subHeading">Explore, Imagine, Create</p>
+
 
         </div>
            
         <div class="postButtons">
 
-        	<!-- Insert code for All Posts & Create Post button here -->
+            <!-- Insert code for All Posts & Create Post button here -->
+            <div id ="parent">
+                 <input class="postButton" type="button" id="allposts" value="All Posts">
+                 <input class="postButton" type="button" onclick="toggleModal1('createpost');" value="Create Post">
+            </div>
 
-        </div>
+
+	    </div>
     </div>
 
     <div>
 
     	<!-- Insert code for Sign In modal here -->
+	<div class="modal" id="signInModal">
+    	<div class="modalContent">
+            <span class="modalClose" onclick="toggleModal('signInModal');">&times;</span>
+            <h1>Welcome Back!</h1>
+            <hr>
+            <form>
+                <label>Username</label>
+                <input type="text" placeholder="Enter Username" required>
+                <label>Password</label>
+                <input type="password" placeholder="Enter Password" required>
+                <input id="signInBtn" type="submit" value="Sign In">
+            </form>
+            <h3>Not a member?
+                <a id="signUpRedirect" onclick="toggleModal('signInModal');toggleModal('signUpModal');" href="javascript:void(0);">Sign Up</a>
+            </h3>
+         </div>
+         </div>
+
+
+	    
 
     </div>
 
     <div>
 
     	<!-- Insert code for Sign Up modal here -->
-
+            <div class="modal" id="signUpModal">
+            <div class="modalContent">
```

Comment:
Please make sure indentation is correct.

Reply:
ok